cns/saas/admin_guide/start/start.adoc

```diff
@@ -1,3 +1,15 @@
 == Get started
 
-To get started, complete the following.
+To get started, complete the following tasks. +
+
+. xref:./create-ns.adoc[Create namespaces]
+
+. xref:./enforcer/enforcer.adoc[Install the Enforcer]
+
+To quickly apply microsegmentation to an application, follow these steps. +
+
+* https://github.com/alexandre-cezar/cns-docs/blob/main/Monitor%20Traffic.adoc[Monitor Network traffic]
```

Comment:
These links are to a private repo

Reply:
Yes they were placeholder for the topics that needed to be added